README.md

```diff
@@ -1,16 +1,16 @@
-# TaigaMobile  
-<a href='https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=300/></a>  
+# TaigaMobile
+<a href='https://play.google.com/store/apps/details?id=io.eugenethedev.taigamobile&utm_source=github'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=300/></a>
+
+This is the **unofficial** android app for the agile project management system [taiga.io](https://www.taiga.io/). It was built with [Jetpack Compose](https://developer.android.com/jetpack/compose), featuring Material You with dynamic colors. The project is still under development and new features will be added in the future.
```

Comment:
`This is the` - i'm not sure that we need `the` here

Reply:
English grammar requires an article here. The "the" is needed for the sentence to be grammatically correct.